env-vars.js

```diff
@@ -29,7 +29,7 @@ config.VIDEO_KEY = env.VIDEO_KEY;
 
 writeFileSync('./src/environments/.env.json', JSON.stringify(config));
 
-const buildSwitches = `--aot --base-href=${config.PUBLIC_PATH} ${
+const buildSwitches = `${config.LUMEER_ENV === 'production' ? '--prod' : '--aot'} --base-href=${config.PUBLIC_PATH} ${
```

Comment:
Isn' the --aot flag deprecated/non-functional at all?

Reply:
It's not. It's just that the AOT should be turned on by default. The update script from Angular CLI puts it to `angular.json` but I can see it hasn't been run here.